include/libzfs_core.h

```diff
@@ -84,16 +85,16 @@ int lzc_send_space(const char *, const char *, enum lzc_send_flags, uint64_t *);
 struct dmu_replay_record;
 
 int lzc_receive(const char *, nvlist_t *, const char *, boolean_t, boolean_t,
-    int);
-int lzc_receive_resumable(const char *, nvlist_t *, const char *, boolean_t,
     boolean_t, int);
+int lzc_receive_resumable(const char *, nvlist_t *, const char *, boolean_t,
```

Comment:
This changes lzc_recv* API, breaking pyzfs tests: we need to update libzfs_core usage in "contrib/pyzfs".

```
/usr/share/zfs/test-runner/bin/test-runner.py  -c /usr/share/zfs/runfiles/linux.run -T pyzfs -i /usr/share/zfs/zfs-tests -I 1
[ 8412.331596] python[17740]: segfault at 7ffcffffffff ip 00007f70ec8fd5f8 sp 00007ffc7cd04010 error 4 in libzfs_core.so.1.0.0[7f70ec8fb000+5000]
Test: /usr/share/zfs/zfs-tests/tests/functional/pyzfs/pyzfs_unittest (run as root) [00:58] [FAIL]

Results Summary
FAIL	   1

Running Time:	00:00:58
Percent passed:	0.0%
Log directory:	/var/tmp/test_results/20180508T002723

root@linux:~# gdb -q `which python` /var/crash/python.17740 
Reading symbols from /usr/bin/python...(no debugging symbols found)...done.

warning: core file may not match specified executable file.
[New LWP 17740]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Core was generated by `python -m unittest --verbose libzfs_core.test.test_nvlist.TestNVList libzfs_cor'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007f70ec8fd5f8 in recv_impl (snapname=0x7f70ebfefb34 "pool.ad3ee684-4d39-4e1a-8da6-0280d7db3d9c/recv@snap1", recvdprops=0x1957e80, localprops=localprops@entry=0x0, origin=0x0, force=B_FALSE, 
    resumable=B_FALSE, raw=B_FALSE, holds=(B_TRUE | unknown: 2), input_fd=-316085704, begin_record=0x7ffcffffffff, cleanup_fd=-321710016, read_bytes=0x7f70ecd318e8, errflags=0x7f70ecd317d0, 
    action_handle=0x7f70ebfc9728, errors=0xa) at libzfs_core.c:705
705			drr = *begin_record;
(gdb) bt
#0  0x00007f70ec8fd5f8 in recv_impl (snapname=0x7f70ebfefb34 "pool.ad3ee684-4d39-4e1a-8da6-0280d7db3d9c/recv@snap1", recvdprops=0x1957e80, localprops=localprops@entry=0x0, origin=0x0, force=B_FALSE, 
    resumable=B_FALSE, raw=B_FALSE, holds=(B_TRUE | unknown: 2), input_fd=-316085704, begin_record=0x7ffcffffffff, cleanup_fd=-321710016, read_bytes=0x7f70ecd318e8, errflags=0x7f70ecd317d0, 
    action_handle=0x7f70ebfc9728, errors=0xa) at libzfs_core.c:705
#1  0x00007f70ec8fe229 in lzc_receive_one (snapname=<optimized out>, props=<optimized out>, origin=<optimized out>, force=<optimized out>, resumable=<optimized out>, raw=<optimized out>, 
    holds=(B_TRUE | unknown: 2), input_fd=-316085704, begin_record=0x7ffcffffffff, cleanup_fd=-321710016, read_bytes=0x7f70ecd318e8, errflags=0x7f70ecd317d0, action_handle=0x7f70ebfc9728, errors=0xa)
    at libzfs_core.c:921
#2  0x00007f70edb55f60 in ffi_call_unix64 () from /usr/lib/x86_64-linux-gnu/libffi.so.6
#3  0x00007f70edb559c8 in ffi_call () from /usr/lib/x86_64-linux-gnu/libffi.so.6
#4  0x00007f70edd63c72 in ?? () from /usr/lib/python2.7/dist-packages/_cffi_backend.x86_64-linux-gnu.so
#5  0x00000000004c9e8b in PyEval_EvalFrameEx ()
#6  0x00000000004e4518 in ?? ()
#7  0x00000000004ccd09 in PyEval_EvalFrameEx ()
#8  0x00000000004c7a59 in PyEval_EvalCodeEx ()
```

Reply:
This was been fixed in the most recent push.